src/custom/components/Button/index.tsx

```diff
@@ -22,20 +22,20 @@ import {
 
 export const ButtonPrimary = styled(ButtonPrimaryMod)`
   // CSS overrides
+  ${({ theme }) => theme.linearGradient}
```

Comment:
This isn't quite obvious that `linearGradient = 'background-image:...'`, maybe call it `bckImageGradient`
Or use like `background-image:   ${({ theme }) => theme.linearGradient}`

Reply:
ok, will do

Reply:
1e4bfbb5faeaa973ea0fb416938044d887435567